Swift/Helpers/FSExtensions/FSE+UIAlertView.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  FSE+UIAlertView.swift
+//  FSHelpers
+//
+//  Created by Sergey Nikolaev on 05.01.16.
+//  Copyright © 2016 FlatStack. All rights reserved.
+//
+
+import UIKit
+
+public extension UIAlertController {
+    
+    func fs_show () {
```

Comment:
That is analog for UIAlertView(...).show()
So I think it's too much
But I will add documentation code here